libs/ledger-live-common/src/hw/connectApp.ts

```diff
@@ -57,6 +66,16 @@ export type ConnectAppRequest = {
   requireLatestFirmware?: boolean;
   outdatedApp?: AppAndVersion;
   allowPartialDependencies: boolean;
+  passDeprecation?: boolean;
+};
+
+export type DeviceDeprecation = {
+  warningScreen: boolean;
```

Comment:
Consider using a different naming for booleans. For instance `"warningScreenVisible"` (if this is what this property means) is much easier to understand.